sprint3/zod.js

```diff
@@ -0,0 +1,49 @@
+import { z } from "zod";
+import isUuid from "is-uuid"
+
+const uuidSchema = z.string().refine(isUuid, {
+  message: "Invalid UUID format",
+});
```

Comment:
isUuid는 함수가 아니라 객체입니다. 지금 상태에서 uuid fomart string을 검증하도록 하면, 아래 에러가 발생합니다.
`TypeError: def.fn is not a function`
현재 코드에서는 아래처럼 함수형태로 변경해서, 특정 uuid버전만 사용하도록 유도할 수 있어요.

```suggestion
const uuidSchema = z.string().refine(isUuid.v4, {
  message: "Invalid UUID format",
});
```

혹은 어떤 버전이든 상관없이 UUID 포맷인지만 검사하고 싶다면
`const uuidSchema = z.string().uuid();`